IoT/vss-extension.json

```diff
@@ -705,6 +729,19 @@
                 "type": "resource",
                 "iconSvgContent": "<svg version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px'  viewBox='0 0 50 50' style='enable-background:new 0 0 50 50;' xml:space='preserve'>  <g>    <path fill='#017ACC' d='M47.7,35H2.3C1.6,35,1,35.6,1,36.3v12.4C1,49.4,1.6,50,2.3,50h45.4c0.7,0,1.3-0.6,1.3-1.3V36.3  C49,35.6,48.4,35,47.7,35z' />    <rect x='42' y='38' fill='#FFFFFF' opacity='0.2' enable-background='new' width='3' height='3' />    <rect x='42' y='44' fill='#FFFFFF' opacity='0.2' enable-background='new' width='3' height='3' />    <path fill='#017ACC' d='M42.5,14.8c0.4-1.1,0.6-2.3,0.6-3.5c0-6.2-5.1-11.2-11.4-11.2c-4.8,0-8.8,2.9-10.5,7c-1.5-1.6-3.7-2.6-6-2.6  C10.6,4.5,7,8.1,7,12.6c0,0.8,0.1,1.6,0.3,2.3c-3.6,1.3-6.1,4.7-6.1,8.7c0,5.1,4.2,9.3,9.4,9.3h29c5.2,0,9.4-4.2,9.4-9.3  C48.9,19.6,46.2,16.1,42.5,14.8z' />    <path fill='#FFFFFF' opacity='0.2' enable-background='new' d='M4.3,28.9c0-4.1,2.7-7.5,6.4-8.9c-0.2-0.7-0.4-1.6-0.4-2.4c0-4.6,3.8-8.3,8.6-8.3c2.5,0,4.8,1,6.3,2.6  c1.8-4.1,6-7.1,11-7.1c2,0,3.9,0.5,5.5,1.3C39.9,2.5,36,0,31.5,0C26.7,0,22.7,2.8,21,6.7c-1.5-1.5-3.7-2.5-6-2.5  c-4.6,0-8.2,3.5-8.2,7.9c0,0.8,0.1,1.6,0.3,2.3C3.5,15.7,1,19,1,22.8c0,2.7,1.3,5.2,3.3,6.9C4.3,29.5,4.3,29.2,4.3,28.9z' />    <path fill='#1E1E1E' opacity='0.2' enable-background='new' d='M41.4,25.1c-0.6-1.4-1.9-2.4-3.5-2.4c-0.5,0-0.9,0.1-1.4,0.3c-0.1,0-0.2,0.1-0.3,0.2l-3.7-3.5  c0.3-0.7,0.4-1.4,0.4-2.1c0-3-2.5-5.5-5.5-5.5c-1.5,0-2.8,0.6-3.8,1.5l-2-1.2c0-0.6,0-1.1-0.2-1.7c-0.6-1.4-1.9-2.4-3.5-2.4  c-0.5,0-0.9,0.1-1.4,0.3c-1.9,0.7-2.9,2.9-2.1,4.8c0.6,1.4,1.9,2.4,3.5,2.4l0,0c0.5,0,0.9-0.1,1.4-0.3c0.3-0.1,0.6-0.3,0.8-0.5  l2,1.2c0,0.4-0.1,0.8-0.1,1.3c0,0.6,0.1,1.2,0.3,1.7l-3.9,2.4c-0.5-0.3-1-0.4-1.6-0.4c-0.4,0-0.8,0.1-1.2,0.2  c-1.6,0.6-2.4,2.5-1.8,4.1c0.5,1.2,1.6,2,3,2l0,0c0.4,0,0.8-0.1,1.2-0.2c1.3-0.5,2-1.7,2-3l4-2.5c0.5,0.4,1.1,0.7,1.7,0.9  l-2.6,15.5c-1.8,0.6-3.1,2.3-3.1,4.3c0,2.5,2,4.5,4.5,4.5s4.5-2,4.5-4.5c0-1.9-1.2-3.5-2.9-4.2l2.6-15.4c0.7-0.2,1.4-0.4,1.9-0.8  l3.7,3.6c-0.2,0.7-0.1,1.5,0.1,2.2c0.6,1.4,1.9,2.4,3.5,2.4c0.5,0,0.9-0.1,1.4-0.3C41.2,29.2,42.2,27,41.4,25.1z' />    <path fill='#FFFFFF' enable-background='new' d='M40.5,25.5C40,24.1,38.4,23.4,37,24c-0.3,0.1-0.6,0.3-0.8,0.5l-4.8-4.6c0.4-0.7,0.7-1.5,0.7-2.3  c0-2.5-2-4.5-4.5-4.5c-1.5,0-2.8,0.7-3.6,1.8l-3.3-2.1c0.2-0.6,0.2-1.2-0.1-1.8C20,9.6,18.4,8.9,17,9.4s-2.1,2.1-1.5,3.5  c0.5,1.4,2.1,2.1,3.5,1.5c0.4-0.2,0.8-0.4,1.1-0.8l3.3,2c-0.2,0.5-0.4,1.1-0.4,1.7c0,0.8,0.2,1.5,0.5,2.1l-5.2,3.2  c-0.6-0.6-1.5-0.8-2.3-0.5c-1.1,0.4-1.7,1.7-1.2,2.8s1.7,1.7,2.8,1.2c1.1-0.4,1.6-1.5,1.3-2.6l5.3-3.3c0.7,0.8,1.6,1.3,2.7,1.4  L24.3,39c-1.8,0.1-3.3,1.6-3.3,3.5s1.6,3.5,3.5,3.5s3.5-1.6,3.5-3.5c0-1.7-1.2-3-2.7-3.4L27.9,22c1.1-0.1,2.1-0.5,2.8-1.3l4.8,4.6  c-0.3,0.7-0.3,1.4-0.1,2.1c0.5,1.4,2.1,2.1,3.5,1.5S41,26.9,40.5,25.5z' />  </g></svg>"
             }
+        },
+        {
+            "id": "service-type-existing-edge-device",
```

Comment:
This one is not used anywhere?

Reply:
yes, this is not used and was already removed by you. My mistake to add it back. I will remove.